### ISSUE TITLE
Add outside collaborators to this repo and project board

### ISSUE THREAD

**Issue by oindrillac** [open-services-group/byon#25]
This working group extends across multiple teams and we would like to add external collaborators to this repo and project (such as ODH team members). By default this is not possible at the moment, so we propose to :-
 
1. Add them as members of the org
2. Put them on a `wg-byon-collaborators` team
3. Add team with write perms to project

Any inputs?

cc: @goern

**Comment by durandom** [comment]
@tumido can ploigos manage external collaborators per repo?

**Comment by tumido** [comment]
Nope, peribolos can't handle that. Presumption is that if you are a regular contributor, who is being assigned issues and what not = you are part of a team. External colaborators are not supported in Peribolos.

Github allows issues to be assigned to more than github org members though, citing all the options below:

1. anyone who has commented on the issue or pull request,
2. anyone with write permissions to the repository,
3. and organization members with read permissions to the repository

https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users#about-issue-and-pull-request-assignees

**Comment by tumido** [comment]
So, if somebody external comments on an issue, we can assign them.

The other options are:
1. either make them org member via peribolos
2. or add them as external members with write permissions to the repo outside of peribolos

**Comment by oindrillac** [comment]
@open-services-group/steering did we reach any consensus on we can approach this?

**Comment by goern** [comment]
I think the root of the issue is, that we host a project on OSG org, but at the same time understand OSG org as something 'for us', so an external contributor as an org member feels strange. Is that a correct interpretation of how we see our org?

So the midterm solution could be:
1. get org-maintenance services implemented for odh org
2. move byon repo to odh org
2.1. accept external contributors as org members on odh org
3. move byon project to odh org
4. have OSG hosting the WG on behalf of ODH

wdyt?

Short term, I would go with Tom's proposed 2. as is seems to be a relatively small number of ppl

**Comment by durandom** [comment]
> Github allows issues to be assigned to more than github org members though, citing all the options below:
> 
> 1. anyone who has commented on the issue or pull request,
> 2. anyone with write permissions to the repository,
> 3. and organization members with read permissions to the repository

Can we manually add external contributors to a repo? And would that make those accounts available in the dropdown?

I'm 👎  for adding other folks to our org

**Comment by goern** [comment]
> I'm -1 for adding other folks to our org

so you are +1 on option 2. of https://github.com/open-services-group/byon/issues/25#issuecomment-1055447355
